src/Controls/src/Core/Platform/ModalNavigationManager/ModalNavigationManager.Android.cs

```diff
@@ -208,9 +219,10 @@ internal class ModalFragment : DialogFragment
 			NavigationRootManager? _navigationRootManager;
 			static readonly ColorDrawable TransparentColorDrawable = new(AColor.Transparent);
 			bool _pendingAnimation = true;
+			bool _presentationCompleted = false;
```

Comment:
I would change there to be `pendingNavigation` and be aligned with `PendingAnimation` logic